### DISCUSSION TITLE
[Question][Hw3] No newline at EOF for example3.expected file

### DISCUSSION THREAD

**Discussion by pacokwon** [prosyslab-classroom/cs348-information-security#524] [Q&A]
While all the other `.expected` files have newline at the end of file, the `test/example3.expected` file does not have a terminating newline. Inspecting the last byte with `tail` does not produce a newline while others do.
```bash
$ tail -c 1 test/example3.expected
5
```

This leads to failure in my testing (only in example3), because the `.output` files have a terminating newline. I think perhaps the `.expected` files should be consistent in this matter!

Thank you.

**Comment by pacokwon** [comment]
@KihongHeo can you please look into this issue? No one seems to address this one.

**Comment by hyunsikjeong** [comment]
@pacokwon I'm not a TA of this course (CS448), but it seems it's a bit impolite to put 🤔 even you're waiting for only 2 hours. TAs are also students and they have their works.
Ask @junseon0-0 for this one.

**Comment by pacokwon** [comment]
@jhs7jhs I guess I was impatient because all the other issues were active but this one didn't seem to get a lot of attention. I certainly didn't mean to be impolite, but I'll edit it out. Thanks for letting me know!

**Comment by Junseon00** [comment]
I just saw it now. I'll put the newline of EOF at that file like others. After fix it I'll announce it. Thanks.

**Comment by pacokwon** [comment]
Thanks a bunch! I appreciate it.
